console/console-init/ui/src/modules/address-space/containers/MessagingToolbarContainer/MessagingToolbarContainer.tsx

```diff
@@ -97,16 +97,23 @@ export const MessagingToolbarContainer: React.FunctionComponent<IMessagingToolba
     setNameInput(undefined);
   };
 
-  const onChangeNameInput = async (value: string) => {
+  const getAddressSpaceForNameORNameSpace = async (
+    value: string,
```

Comment:
All the 'value's in all the queries are actually search values, how about renaming 'value' to 'searchValue'?

Reply:
i think value is fine for now.